src/learn/Learn.tsx

```diff
@@ -10,32 +10,46 @@ import LearnSectionMenu from './LearnSectionMenu';
 
 import * as css from './Learn.m.css';
 
+interface Guide {
+	name: string;
+	directory?: string;
+	repo?: string;
+	branch?: string;
+}
+
 interface LearnProperties {
 	guideName: string;
 	pageName: string;
 	url?: string;
+	repo?: string;
+	branch?: string;
 }
 
 const factory = create({ theme, i18n }).properties<LearnProperties>();
 
-export const guides = [
-	'Overview',
-	'Creating Widgets',
-	'Middleware',
-	'Building',
-	'I18n',
-	'Styling',
-	'Stores',
-	'Routing',
-	'Testing'
+export const sources = {
+	framework: {
+		repo: 'dojo/framework',
+		branch: 'v6'
+	}
+};
+
+export const guides: Guide[] = [
+	{ name: 'Overview' },
+	{ name: 'Creating Widgets' },
+	{ name: 'Middleware' },
+	{ name: 'Building' },
+	{ name: 'I18n' },
+	{ name: 'Styling' },
+	{ name: 'Stores' },
+	{ name: 'Routing' },
+	{ name: 'Testing' }
 ];
 
 export default factory(function Learn({ properties, middleware: { theme, i18n } }) {
-	const { guideName, pageName, url } = properties();
+	const { guideName, pageName, url, repo = sources.framework.repo, branch = sources.framework.branch } = properties();
```

Comment:
Repo and branch do not appear to be used elsewhere, since `repo` and `branch` variables are deconstructed from the guide object and then only used in that scope.

Reply:
I'm not sure what you mean. On this line, repo/branch is used to display the active content but elsewhere repo/branch is used (from the full list of guides) with a loop to display navigation.

Reply:
My point is that `repo` and `branch` get declared twice (here and line 67). They are only used in the scope of the variables declared on line 67, thus these two variables from the properties would never get used.

Reply:
Okay I see where it is being used now. Carry on!